Dockerfile

```diff
@@ -74,5 +74,5 @@ CMD [ "api" ]
 FROM application AS testing
 
 USER 0
-RUN pip install -r requirements-test.txt
+RUN pip install -r test-requirements.txt
```

Comment:
Why changing the file name?

Reply:
A bit of a nitpick but I prefer the requirements-* naming convention and we use it in most Sentry repos.

Reply:
Renamed them.